src/AccessibilityContext.tsx

```diff
@@ -9,7 +9,11 @@ export const AccessibilityContext = createContext<AccessibilityState>({
   isScreenReaderEnabled: false,
 });
 
-export const AccessibilityContextProvider: React.FC = ({children}) => {
+interface Props {
+  children: React.ReactNode;
+}
+
+export const AccessibilityContextProvider = ({children}: Props) => {
```

Comment:
@rosvik @gorandalum I was upgrading the react-native that now uses react 19 and there are some changes as listed here https://react.dev/blog/2024/04/25/react-19-upgrade-guide, seems like this is the way to go now, what do you think?